unison-src/transcripts/resolve.output.md

```diff
@@ -227,18 +228,19 @@ We still have a remaining _name conflict_ since it just so happened that both of
 
   ❓
   
-  These terms have conflicting definitions: foo
+  The term foo has conflicting definitions:
+    1. foo#8e68dvpr0a
+    2. foo#jdqoenu794
```

Comment:
I added the conflicts to the `todo` output directly rather than requiring a separate `view` command since it seems pertinent to have them.

Reply:
Contrasting this to the previous case, `foo` doesn't appear to be a numbered output here, although it could be (it would just be the name in this case, with no hash qualifier).

Reply:
Yeah I avoided numbering it here since using just the name would be ambiguous, note that this one is due to a conflict in the namespace, not a conflict in the patch, so AFAIK you wouldn't use `replace` here right?